zerver/tests/test_message_edit.py

```diff
@@ -1068,33 +1068,271 @@ def test_notify_old_thread_move_message_to_stream(self) -> None:
         messages = get_topic_messages(user_profile, new_stream, "test")
         self.assertEqual(len(messages), 3)
 
-    def test_move_message_to_stream_to_private_stream(self) -> None:
-        user_profile = self.example_user("iago")
+    def test_move_message_to_stream_public_to_private_stream_not_shared_history(self) -> None:
```

Comment:
These names feel a bit awkward; how about `test_move_message_from_public_to_private_not_shared_history` as the pattern?